packages/decdk/README.md

```diff
@@ -269,8 +271,7 @@ deCDK can deconstruct APIs that adhere to the standards defined by __awslint__ a
 
 As you might have observed, whenever users need to reference a type in deCDK templates they are required to reference the fully qualified name (e.g. `@aws-cdk/aws-s3.Bucket`). We can obvsiouly come up with a more concise way to reference these types, as long as it will be possible to deterministically translate back and forth.
 
-### Misc
+### Special Types
 
-- `iam.PolicyDocument` is be tricky since it utilizes a fluent API. We need to think whether we want to revise the PolicyDocument API to be more compatible or add a utility class that can help.
+`iam.PolicyDocument` is be tricky since it utilizes a fluent API. We need to think whether we want to revise the PolicyDocument API to be more compatible or add a utility class that can help.
```

Comment:
is be

Reply:
![](https://d2b8crmh47yoxi.cloudfront.net/wp-content/uploads/2014/08/blog-lol-cat.jpg)